changelog/fragments/00-template.yaml

```diff
@@ -1,45 +1,5 @@
-# entries is a list of entries to include in
-# release notes and/or the migration guide
 entries:
   - description: >
-      Description is the line that shows up in the CHANGELOG. This
-      should be formatted as markdown and be on a single line. Using
-      the YAML string '>' operator means you can write your entry
-      multiple lines and it will still be parsed as a single line.
-
-      If an operator type's plugin is being changed then prefix the description with
-      "(<operator language>/<plugin version>)", ex. "(go/v2)".
-      If an operator type's runtime is being changed, use then prefix the description
-      with "For <operator language>-based operators,", ex. "For Go-based operators".
-
-    # kind is one of:
-    # - addition
-    # - change
-    # - deprecation
-    # - removal
-    # - bugfix
-    kind: ""
-
-    # Is this a breaking change?
+      Added `watch` permission for `kubebuilder:rbac:groups=core,resources=pods` in Go Operator Tutorial.
+    kind: "bugfix"
```

Comment:
Just realized a problem here, this is making a modification to our template that we use for creating changelogs. Would you mind reverting this change and creating a separate file for the changelog? 

Thanks!